libraries/shared/storage/types/value.go

```diff
@@ -22,6 +22,7 @@ type ValueType int
 
 const (
 	Uint256 ValueType = iota
+	Uint8
```

Comment:
I think we need to add a branch to the storage decoder so that we'll properly handle this new type